app/models/user.rb

```diff
@@ -74,6 +74,15 @@ def to_s
     "#{email}"
   end
 
+  def editor_practices
+    editor_list = PracticeEditor.where(user_id: id)
```

Comment:
⚠️ Please let me know if there is a better way to get all the user's practices via the PracticeEditor relationship 🤔

Reply:
Here's a simpler way. Not sure about better, but try it out and let me know what ya think!

```
PracticeEditor.where(user: self).collect { |pe| pe.practice }
```

Reply:
ooo nice TIL! thank you!